Documentation/supermuc.rst

```diff
@@ -22,8 +22,20 @@ Here, we described the procedure to set up such port forwarding.
      RemoteForward ddddd github.com:22
 
 where ddddd is an arbitrary 5-digital port number.
+  
+2. Confirm the creation of the new ssh profile by typing below command:
+
+::
+
+  bash
```

Comment:
I don't see how this command confirms the creation of a new ssh profile?

Reply:
Ah I get it, with ssh config this 'bash' is not needed.